hilo.rb

```diff
@@ -1,2 +1,20 @@
-number = rand(1..100)
-puts number 
\ No newline at end of file
+random_number = rand(1..100)
+puts "Enter a number between 1 and 100, then press enter to submit your guess. You have 6 chances to guess the correct number"
+input = gets.chomp
+guess = input.to_i
+count = 0
+while count < 5
+    count = count + 1
+    if guess.to_s != input
+        puts "Your guess is not a whole number. Try again"
+    elsif guess > 100
```

Comment:
I think it should be `guess > random_number` or `guess < random_number` 'cause the player needs to know if his guess is within a range or not.

Reply:
oh I see, I read the instructions wrong! I thought the guess had to be between 1 and 100 but that's the random number! Thanks, I'll change that now